update readme with guide on project submissions (#36)
aaif/project-proposals: README.md

## How to Apply

> 1. Check out the guide on [How to Submit your Project to the AAIF](https://aaif.io/blog/how-to-submit-your-project-to-the-aaif/) for more details on what we're looking for in a project.

Author: ```suggestion
1. Check out the guide on [what we're looking for and why you may want to submit your project](https://aaif.io/blog/how-to-submit-your-project-to-the-aaif/).
```
